src/service/webview.service.ts

```diff
@@ -430,7 +431,7 @@ export class WebviewService {
     gistSelectionPanel.webview.html = content;
     gistSelectionPanel.webview.onDidReceiveMessage(async message => {
       if (!message.close) {
-        const extSettings = await state.commons.GetSettings();
+        const extSettings = state.commons.GetSettings();
```

Comment:
`await` is removed?

Reply:
Same to https://github.com/shanalikhan/code-settings-sync/pull/1257#discussion_r495887677